.github/workflows/ci.yml

```diff
@@ -48,6 +48,11 @@ jobs:
           - name: basic
             config_path: "ci/configs/basic.yml"
             skips: --skip-javascript
+          - name: basicauth
+            config_path: "ci/configs/basicauth.yml"
```

Comment:
we don't need a new config for this since it's just the env variables that are different